src/index.js

```diff
@@ -246,9 +246,8 @@ const DealerLocatorWrapper = styled.div`
   grid-template-columns: calc(${(props) => props.theme.menuSlideoutWidth}) 1fr;
   
   @media (max-width: ${(props) => props.theme.sideBySideLayoutBreakpoint}) { 
-     grid-template-areas:
-      "search"
-      "map";
+     grid-template-areas: ${(props) => props.visible ? `"search map" "map map"` : `"search" "map"`};
+     grid-template-columns: 75vw 25vw;
```

Comment:
Is there any way to do this without basing width on vw? I'm still holding out some hope that the dealer-locator module should be able to work in a variety of layouts, but if you set widths like this then it would never work in any context where it's not in a full width container.

Reply:
Seems to also work with percentages.

Reply:
Updated to use a percentage of the column track.